From af8e3e1f5a0c491d4a470238d54feac1ba1be30c Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 14 Sep 2026 21:28:59 +0800
Subject: [PATCH 1/2] AOSCOS: RETRO: fix(khelpmenu): disable Handbook entry

KHelpCenter is practically a browser and uses an enormous amount of RAM
out of our ~128MiB budget.

So we just don't ship it - we have smartphones now.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 tdeui/khelpmenu.cpp | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/tdeui/khelpmenu.cpp b/tdeui/khelpmenu.cpp
index a274ec737..64615b69d 100644
--- a/tdeui/khelpmenu.cpp
+++ b/tdeui/khelpmenu.cpp
@@ -125,14 +125,6 @@ TDEPopupMenu* KHelpMenu::menu()
     connect( mMenu, TQ_SIGNAL(destroyed()), this, TQ_SLOT(menuDestroyed()));
 
     bool need_separator = false;
-    if (kapp->authorizeTDEAction("help_contents"))
-    {
-      mMenu->insertItem( BarIcon( "contents", TDEIcon::SizeSmall),
-                     TQString(i18n( "%1 &Handbook" ).arg( appName)) ,menuHelpContents );
-      mMenu->connectItem( menuHelpContents, this, TQ_SLOT(appHelpActivated()) );
-      mMenu->setAccel( TDEStdAccel::shortcut(TDEStdAccel::Help), menuHelpContents );
-      need_separator = true;
-    }
 
     if( mShowWhatsThis && kapp->authorizeTDEAction("help_whats_this") )
     {
-- 
2.55.0

